src/main/webapp/WEB-INF/classes/.gitignore

```diff
@@ -0,0 +1 @@
+/META-INF/
```

Comment:
I can't figure when i modified the file src/main/webapp/WEB-INF/classes/.gitignore, but i found our from the git log that if i have to undo changes to this file , i have to revert the whole commit and refix the error at fresh . Should i go ahead and revert the whole commit ?